lib/view.js

```diff
@@ -25,6 +25,17 @@ module.exports = Vue.extend({
             e.stopPropagation();
             e.preventDefault();
             return false;
+        },
+        createAppFromTemplate: function (templateId) {
+            var self = this;
+            self.$root.isReady = false;
+            var app = self.$root.storage.createApp({
+                template: templateId
+            });
+            setTimeout(function () {
```

Comment:
This is just a visual thing, so the app being created feels more like a deliberate action. It's synchronous because if we make it async/in a callback, it won't work in an offline state